department-of-veterans-affairs/abd-vro
# Saving evidence count summary as json object

# MCP-1887 Assessment Result Population

## Description



### What was the problem?

We need to be able to save the Evidence Summary information from full-health-data-assessment into the Assessment Result table in the DB.

### How does this fix it?

Adjusted SaveToDbService to save evidence summary as JSON in assessment result table

### Jira Tickets



- [MCP-1887](https://amida.atlassian.net/browse/MCP-1887)

## How to test this PR

- Execute full-health-data-assessment in swagger
- Verify the evidence summary is in the assessment_result table in the Postgres DB

## Reminders

- Review [Pull-Requests](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests) guidelines
[Roadmap](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Roadmap) wiki pages
